RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -306,6 +306,11 @@ private void BindQuickFixTypes(IEnumerable<Assembly> assemblies)
             foreach (var quickFix in quickFixes)
             {
                 Bind<IQuickFix>().To(quickFix).InSingletonScope();
+                if (quickFix.Name.Equals("AssignedByValParameterMakeLocalCopyQuickFix"))
```

Comment:
Wait a sec... this is a `Factory` type. There shouldn't even be a need to have a concrete implementation.